app/scripts/dimApp.i18n.js

```diff
@@ -99,19 +99,19 @@
           General: "Allgemein",
           Postmaster: "Poststelle",
           Equip: "Ausrüsten",
-          Store: "Speichern",
+          Store: "In den Tresor legen",
```

Comment:
Is there a translation that doesn't involve the vault, something generic that means "to hold on to" (e.g. hoard)